lib/membrane_vpx/encoder/vp9_encoder.ex

```diff
@@ -26,6 +26,13 @@ defmodule Membrane.VP9.Encoder do
                 If set to `:auto` the deadline will be calculated based on the framerate provided by
                 incoming stream format. If the framerate is `nil` a fixed deadline of 10ms will be set.
                 """
+              ],
+              target_bitrate: [
+                spec: pos_integer(),
+                default: 1000,
+                description: """
+                Gives the encoder information about the target bitrate (in kb/s).
+                """
```

Comment:
Please put here some examples, what range of values is reasonable for a video with 2560 x 1440, 1280 x 720 or 640 x 360, to give an impression, what bitrate value is low for a specific video size and what bitrate is high.

Do it here and in VP8 Encoder.

Beyond this, I think that `default: 1000` is not a good idea - it will cause having a good video quality, when the video size is small and bad video quality, when the video size is too big. Default bitrate may vary depending on the size of the input video.